tests/suites/test_suite_psa_crypto.data

```diff
@@ -995,6 +995,10 @@ PSA MAC verify: CMAC-AES-128, truncated to 4 bytes
 depends_on:MBEDTLS_CMAC_C:MBEDTLS_AES_C
 mac_verify:PSA_KEY_TYPE_AES:"2b7e151628aed2a6abf7158809cf4f3c":PSA_ALG_TRUNCATED_MAC(PSA_ALG_CMAC, 4):"6bc1bee22e409f96e93d7e117393172aae2d8a571e03ac9c9eb76fac45af8e5130c81c46a35ce411":"dfa66747"
 
+PSA MAC setup: incompatible key MD for HMAC
```

Comment:
It's not-supported, not incompatible. And it's relevant that this is with a long key (longer than the block size).

> PSA MAC setup: algorithm known but not supported, long key

Would be good to also have a test case with a short key (less than the block size), for completeness.